### PR TITLE
Add PHP-CS check

### PR DESCRIPTION
**Pull request by nupplaphil**

### CODE REVIEW

**Comment by nupplaphil** [comment]
https://github.com/friendica/friendica-addons/pull/1123/commits/622ec88fe302046de20f7eaeaa542aa1d6fbacd5 <-- with wrong PHP files included
https://github.com/friendica/friendica-addons/pull/1123/commits/95737984a0c2a482546063e3b63361ac97c1802e <-- wrong PHP files deleted :-)

I'm done! :-)

File: .drone.yml [hunk 1]
```diff
@@ -32,3 +32,40 @@ steps:
...
+
+clone:
+  disable: true
+
+trigger:
+  event:
+    - pull_request
+
+steps:
+  - name: clone friendica base
```
Comment (MrPetovan) [inline .drone.yml]:
```suggestion
  - name: Clone friendica base
```

File: .drone.yml [hunk 2]
```diff
@@ -32,3 +32,40 @@ steps:
...
+    image: alpine/git
+    commands:
+      - git clone https://github.com/friendica/friendica.git .
+      - git checkout $DRONE_COMMIT_BRANCH
+  - name: clone friendica addon
```
Comment (MrPetovan) [inline .drone.yml]:
```suggestion
  - name: Clone friendica addon
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]
Let's spruce the step names up a bit, shall we?

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**